### PR TITLE
Fixed:Issue#2023 bookmarking incorrectly

### PR DESCRIPTION
**Pull request by NayanKhedkar**
Unexpected bookmarking when trickle enabled.

### CODE REVIEW

**Comment by NayanKhedkar** [comment]
issue:[#2023](https://github.com/adaptlearning/adapt_framework/issues/2023)

File: js/trickleView.js [hunk 1]
```diff
@@ -41,7 +44,9 @@ define([
         isElementEnabled: function() {
             var trickle = Adapt.trickle.getModelConfig(this.model);
             if (!trickle) return false;
-
+            
+            if(this.model.get(this.completionAttribute)) return false;
```
Comment (oliverfoster) [inline js/trickleView.js]: missing space

File: js/trickleView.js [hunk 2]
```diff
@@ -5,6 +5,8 @@ define([
     var TrickleView = Backbone.View.extend({
 
         isSteplocked: false,
+        
+        completionAttribute : "_isComplete",
```
Comment (oliverfoster) [inline js/trickleView.js]: set this to null as it gets overwritten immediately

**Comment by chetan-hajare** [comment]
I have tested with latest framework (3.1.0). Checking with framework of version 2.1.2.

**Comment by chetan-hajare** [comment]
I have tested on framework of version 2.1.2.  [This ](https://github.com/adaptlearning/adapt_framework/issues/2023#issuecomment-400808278)issue occurs but not due to above changes but due to compatibility issue of trickle extension itself. When I made those changes in the older version of trickle which is compatible with older framework (2.1.2), it works fine. I would suggest to merge this PR as it works fine on latest framework.

**Comment by loudoweb** [comment]
@chetan-hajare So do you suggest me to use an older version of trickle with your fix?

**Comment by chetan-hajare** [comment]
This would be helpful if could test it in the above mentioned way.

**Pull request merged**